index.rst

```diff
@@ -6,24 +6,474 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
-
-   **This technote is a work-in-progress.**
+.. Make in-text citations with: :cite:`bibkey`.
```

Comment:
Note sure where to put this comment, so I'll place it here at the top: when I built the page, the LSWT logo that displays for me ta the upper left corner says "Large Synoptic Survey Telescope". This should be updated.

Reply:
Square team is updating the template for distribution